source/multigrid/mg_transfer_matrix_free.cc

```diff
@@ -112,7 +117,20 @@ void MGTransferMatrixFree<dim,VectorType>::build
                                                    dirichlet_indices,
                                                    weights_unvectorized,
                                                    this->copy_indices_global_mine,
-                                                   this->ghosted_level_vector);
```

Comment:
i fixed this issue in https://github.com/dealii/dealii/pull/4667/commits/f350b1e5f2fb2324e8e035970065ed9fcf13ed40 So the current state of the PR should pass all MG/matrix-free tests